src/dotnet-interactive-vscode-common/src/commands.ts

```diff
@@ -139,6 +139,9 @@ export function registerKernelCommands(context: vscode.ExtensionContext, clientM
             await vscode.commands.executeCommand('dotnet-interactive.stopCurrentNotebookKernel', notebook);
             const _client = await clientMapper.getOrAddClient(notebook.uri);
             restartCompletionSource.resolve();
+            if (isAzureDataStudio(context)) {
```

Comment:
Negate this; it should only execute if _not_ ADS.

Reply:
ops!